src/components/Map/data.ts

```diff
@@ -0,0 +1,25 @@
+type Source = string;
+export const MapData: Record<string, Source> = {
```

Comment:
Instead of having the images in the repository, is there a way to use a sync folder for these instead?  Downloading the maps once with a version number and hosting that sync folder instead?

This gets quite specifically into adding data files into repositories